roles/v2/mining-proxy/proxy-config.toml

```diff
@@ -1,5 +1,7 @@
-upstreams = [{ address = "127.0.0.1", port = 34254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}]
-listen_address = "0.0.0.0"
+upstreams = [{ channel_kind = "ExtendedWithNegotiator",  address = "127.0.0.1", port = 34254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}]
```

Comment:
Is there user docs on what these do? What about docs on how to use/configure/run JN?